website/content/en/docs/overview/_index.md

```diff
@@ -88,6 +88,10 @@ By project type (replace `${IMAGE_VERSION}` with base image version in your proj
 [client-go]:https://github.com/kubernetes/client-go
 [client-go-compat]:https://github.com/kubernetes/client-go#compatibility-matrix
 
+## OLM version compatibility
+
+Operator SDK officially supports the latest 3 versions of OLM present at the time of SDK's release. These versions of OLM manifests are packaged with the SDK binary in the form of `bindata` to support offline installations of OLM with [`operator-sdk olm`][olm-install-cmd] command.
```

Comment:
Nit: OLM images still need to be pulled so packaging manifests in bindata isn't "offline installation", per-se, but low-latency
```suggestion
Operator SDK officially supports the latest 3 versions of OLM present at the time of a given Operator SDK release. These versions of OLM manifests are packaged with the SDK binary in the form of `bindata` to support low-latency installations of OLM with [`operator-sdk olm install`][olm-install-cmd]. Any other version installed with this command may work but is not tested nor officially supported.
```